diff --git a/node_modules/next/dist/client/normalize-trailing-slash.js b/node_modules/next/dist/client/normalize-trailing-slash.js
index 27a457d..048c473 100644
--- a/node_modules/next/dist/client/normalize-trailing-slash.js
+++ b/node_modules/next/dist/client/normalize-trailing-slash.js
@@ -11,7 +11,7 @@ const normalizePathTrailingSlash = (path)=>{
     }
     const { pathname , query , hash  } = (0, _parsePath).parsePath(path);
     if (process.env.__NEXT_TRAILING_SLASH) {
-        if (/\.[^/]+\/?$/.test(pathname)) {
+        if (pathname.endsWith('.html/')) {
             return `${(0, _removeTrailingSlash).removeTrailingSlash(pathname)}${query}${hash}`;
         } else if (pathname.endsWith('/')) {
             return `${pathname}${query}${hash}`;
